src/App.js

```diff
@@ -0,0 +1,46 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { ToastContainer } from 'react-toastify';
+import LoginPage from './pages/LoginPage';
+import ProtectedRoute from './pages/ProtectedRoute';
+import './App.css';
+import 'react-toastify/dist/ReactToastify.css';
+import Vehicles from './pages/Vehicles';
+import ReserveForm from './pages/ReserveForm';
+import Reservations from './pages/Reservations';
+import AddVehicle from './pages/AddVehicle';
+import RemoveVehicle from './pages/RemoveVehicle';
+import VehicleDetails from './pages/VehicleDetails';
+import VehicleReserve from './pages/VehicleReserve';
+
+function App() {
```

Comment:
- [ ] The mobile implementation home page is overflowing.  Kindly touch up your styling so it works as intended:

![image](https://github.com/HERMON-1995/book_vehicle_rental-frontend/assets/57421823/df784f67-bdbd-4731-9c3f-4e1992f85dd6)

If I open the menu, it comes transparent:

![image](https://github.com/HERMON-1995/book_vehicle_rental-frontend/assets/57421823/9d538fc4-b197-4941-9bfd-a58e1b16a18e)

All of this is happening in the cars list (homepage) section.

Reply:
Thanks for pointing this out. We will work on it.